pkg/controller/migmigration/validation.go

```diff
@@ -36,6 +36,7 @@ const (
 	VeleroFinalRestorePartiallyFailed  = "VeleroFinalRestorePartiallyFailed"
 	DirectImageMigrationFailed         = "DirectImageMigrationFailed"
 	StageNoOp                          = "StageNoOp"
+	MigNoOp                            = "MigrateNoOp"
```

Comment:
Suggest renaming this to `MigrateNoOp` to be more specific since `mig` is prefixed to a bunch of stuff.